test/jdk/javax/naming/spi/providers/InitialContextTest.java

```diff
@@ -243,9 +261,15 @@ private static boolean verifyOutput(String output, String fqn) {
 
     private static void jar(Path jarName, Path jarRoot) {
         String jar = getJDKTool("jar");
-        ProcessBuilder p = new ProcessBuilder(jar, "cf", jarName.toString(),
-                "-C", jarRoot.toString(), ".");
-        quickFail(run(p));
+        String [] commands = {jar, "cf", jarName.toString(),"-C", jarRoot.toString(), "."};
+        try {
+            OutputAnalyzer outputAnalyzer = ProcessTools.executeCommand(commands);
+            if(outputAnalyzer.getExitValue() != 0) {
+                throw new RuntimeException(outputAnalyzer.getOutput());
+            }
+        } catch (Exception ex) {
+            throw new RuntimeException(ex.getMessage());
```

Comment:
This and other similar places in this PR will end up swallowing the underlying exception. It would better to just have the method have a `throws Exception` or change this to:

```java
throw new RuntimeException(ex);
```

Reply:
Thanks. Implemented in this commit.